cmd/compile_x64.bat

```diff
@@ -0,0 +1,6 @@
+@echo off
+if EXIST "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community" (
+  "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars64.bat" && cmd\getopt.bat %* && set ARCH=x64&& cmd\make_php.bat
+) else if EXIST "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise" (
+  "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat" && .\cmd\getopt.bat %* && set ARCH=x64&& .\cmd\make_php.bat
+)
```

Comment:
These I simply used the system PATH for and assume the machine has been setup with a PATH to use the correct tooling. I Don't try to own that detail.

Reply:
What you not seeing/realizing everything is setup to be local to the folder you're in. The scripts folder itself can be zipped and extracted, and all building starts there.

The scripts actually downloading PHP if not present in folder current setup.

Reply:
so in theory a benefit of this approach is that I could run parallel builds without setting up separate users, or messing with the PATH variable.

Is that common for windows development?

Reply:
What? The whole thing is customized from there scripts, the reason it works. So NO! It's not following there current system steps. 

And it still seems, that is the part you not realizing, you have to write/edit there steps over. You haven't run my build setup or anything, don't see how you can comment on anything. You don't use Windows daily, otherwise you realize why it's the way it is.

Reply:
Have you considered you are particularly bad at explaining yourself because of these hysterical outbursts?